packages/cli/src/config.js

```diff
@@ -31,6 +31,7 @@ export type TransformConfig = {
   ...CliConfig,
   input: string,
   output: string,
+  cachePath?: string,
```

Comment:
```suggestion
  cachePath?: string = getDefaultCachePath(),
```

Reply:
i believe typescript doesn't allow defaults in a type definition, but could create a factory function to set the default if that's preferable

Reply:
This is a mistake, it should be:

```
cachePath: string = getDefaultCachePath(),
```

The `?` is not needed because it is implied when the default value is provided.